File: LeetCode/Easy/2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int ret = 0;
        
        for (String v : sentences) {
            int temp = 0;
            for (char c : v.toCharArray())
                if (c == ' ') ++temp;
            
            if (ret < temp) ret = temp;
        }

        return ret + 1;
    }
}

File: LeetCode/Easy/2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.go
func mostWordsFound(sentences []string) int {
    ret := 0
        
    for _, v := range sentences {
        temp := 0
        for _, c := range v {
            if c == ' ' { temp++ }
        }
        if ret < temp { ret = temp }
    }

    return ret + 1
}